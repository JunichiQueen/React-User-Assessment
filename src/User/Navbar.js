import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";

class Navbar extends Component{
    render(){
        return(
            <div>
                <ul>
                    <li><Link to="/">Home</Link></li>
                    <li><Link to="/userlist">User List</Link></li>
                    <li><Link to="/login">Login</Link></li>
                </ul>
            </div>
        )
    }
}

export default Navbar;