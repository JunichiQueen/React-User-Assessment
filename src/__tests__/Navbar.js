import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
import Navbar from '../User/Navbar';
import { shallow } from "enzyme";
import renderer from 'react-test-renderer';
import "../setupTests";

it('renders without crashing', () => {
    shallow(<Router><Navbar /></Router>);
});
  
it('snapshot app', () => {
    const wrapper = renderer.create(<Router><Navbar /></Router>).toJSON();
    expect(wrapper).toMatchSnapshot();
});