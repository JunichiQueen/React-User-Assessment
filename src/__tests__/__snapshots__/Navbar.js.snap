// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot app 1`] = `
<div>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/userlist"
        onClick={[Function]}
      >
        User List
      </a>
    </li>
    <li>
      <a
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </li>
  </ul>
</div>
`;
